source/addins/ArcMapAddinVisibility/ViewModels/TabBaseViewModel.cs

```diff
@@ -25,6 +25,8 @@
 using VisibilityLibrary.ViewModels;
 using VisibilityLibrary.Models;
 using ArcMapAddinVisibility.Models;
+using CoordinateConversionLibrary.Models;
```

Comment:
Could have also fully-qualified just this below to avoid fully qualifying everything - but no biggie